api/GPUDevice.json

```
@@ -703,6 +703,47 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "descriptor_bindGroupLayouts_accepts_null_values": {
```

Looks like I didn't apply the [cited guideline](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/README.md#parameters-and-parameter-object-features) properly:

```suggestion
        "descriptor_bindGroupLayouts_parameter_accepts_null_values": {
```